pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -500,6 +496,10 @@ func trimBackingFileName(fileName string) string {
 	return backingFilePattern.ReplaceAllString(fileName, ".vmdk")
 }
 
+func getDiskFullPath(disk ova.Disk) string {
+	return disk.FilePath + "/" + disk.Name
```

Comment:
so this will result in something like `/folder/my.ova/myDisk`? in that case maybe it would be better to change it with something like `/folder/my.ova::myDisk`?